content/docs/command-reference/import-url.md

```diff
@@ -200,6 +206,11 @@ produces a regular stage in `dvc.yaml`.
 
 - `--meta key=value` - custom metadata to add to the data.
 
+- `--version-aware` - capture [cloud versioning] information (supported for
+  certain cloud storage providers). By default, DVC will automatically do so
+  only if the `url` contains a valid cloud versioning ID. Otherwsie, with this
+  flat DVC will import the latest version of the file.
+
 - `-h`, `--help` - prints the usage/help message, and exit.
```

Comment:
I just noticed that this sentence is wrong hah. Should update everywhere (separate PR) to

```suggestion
- `-h`, `--help` - print the usage/help message and exit.
```